skorch/callbacks/scoring.py

```diff
@@ -5,8 +5,8 @@
 from functools import partial
 
 import numpy as np
-from sklearn.metrics.scorer import (
-    check_scoring, _BaseScorer, make_scorer)
+from sklearn.metrics import make_scorer, check_scoring
+from sklearn.metrics.scorer import _BaseScorer
```

Comment:
Most likely needs a version check to avoid the deprecation warning:

```
if version.parse(sklearn.__version__) >= version.parse('0.22.0'):
    from sklearn.metrics._scorer import _BaseScorer
else:
    from sklearn.metrics.scorer import _BaseScorer
```